src/Pages/ChannelRequests.razor

```diff
@@ -141,8 +142,43 @@
                             }
                         </FieldHelp>
                     </Validation>
+                    <div class="mb-3">
+                        <Button Color="Color.Primary" Disabled="@(!_selectedWalletId.HasValue)" Clicked="@OpenCoinSelectionModal">Select Coins</Button> or use
+                        <Button Color="Color.Primary" Disabled="@(SelectedUTXOs.Count == 0)" Clicked="@ClearSelectedUTXOs">Default Coin Selection</Button>
+                    </div>
+                    @if (_selectedWalletId.HasValue && SelectedUTXOs.Count > 0)
+                    {
+                        <span class="text-danger">Selected @(SelectedUTXOs.Count) UTXOs, this is a changeless operation</span>
+                    }
+                    else
+                    {
+                        <span class="text-danger">Default coin selection strategy selected</span>
+                    }
                 </EditTemplate>
             </DataGridColumn>
+            @if (Constants.FEE_SELECTION_ENABLED)
```

Comment:
Added this because started working on it but will finish it in another branch to not clutter this one with too much changes